gamestonk_terminal/cryptocurrency/prediction_techniques/pred_api.py

```diff
@@ -0,0 +1,3 @@
+"""Pred context API."""
+
+# Currently there is no need for this API
```

Comment:
Is there anything missing that the pred api can't be implemented the same way as in the stocks context?

If yes - can you add a TODO comment so that we don't forget to refactor and implement the api?